src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from 'react'
+import { View, TouchableWithoutFeedback } from 'react-native'
+import { useStyles } from '../../../theme'
+import { Box } from '../../structure'
+import { BodyText } from '../../text'
+import { shameStyles } from '../../../theme/shame-styles'
+import { Icon } from '../../images-and-icons'
+
+export interface CheckboxProps {
+  /**
+   * Label to display next to the Checkbox.
+   */
+  label: string
+  /**
+   * True if selected.
+   */
+  checked: boolean
+  /**
+   * Called when selection state changes. Should propagate change to `checked` prop.
+   */
+  onChange: (checked: boolean) => void
+  /**
+   * Additional text to aid in use.
+   */
+  helpText?: string
+}
```

Comment:
See [this comment](https://github.com/hansgx/Cubes/pull/13#discussion_r445074065)

Reply:
Done :wink: